Framework/DataHandling/src/LoadILLTOF2.cpp

```diff
@@ -80,6 +81,7 @@ void LoadILLTOF2::init() {
 void LoadILLTOF2::exec() {
   // Retrieve filename
   const std::string filenameData = getPropertyValue("Filename");
+  m_convertToTOF = getProperty("convertToTOF");
```

Comment:
I suggest you just declare this bool in the scope of this method as it is not used elsewhere, and therefore drop the m_ from the name